File: PHP/Booking/ProtectionInformation.php
<?php
namespace LTD_API;

class ProtectionInformation
{
    private $protectedTicketsCount;
    private $totalProtectionPrice;
	
    public function getProtectedTicketsCount()
    {
        return $this->protectedTicketsCount;
    }
    public function setProtectedTicketsCount($protectedTicketsCount)
    {
        $this->protectedTicketsCount = $protectedTicketsCount;
    }

    public function getTotalProtectionPrice()
    {
        return $this->totalProtectionPrice;
    }
    public function setTotalProtectionPrice($totalProtectionPrice)
    {
        $this->totalProtectionPrice = $totalProtectionPrice;
    }
}

File: Java/Booking/ProtectionInformation.java
package LTD_API;

import java.util.List;

public class ProtectionInformation
{
    private int protectedTicketsCount;
	private decimal totalProtectionPrice;
	
	public int getProtectedTicketsCount()
    {
        return protectedTicketsCount;
    }
    public void setProtectedTicketsCount(int protectedTicketsCount)
    {
        this.protectedTicketsCount = protectedTicketsCount;
    }

    public decimal getTotalProtectionPrice()
    {
        return totalProtectionPrice;
    }
    public void setTotalProtectionPrice(decimal totalProtectionPrice)
    {
        this.totalProtectionPrice = totalProtectionPrice;
    }
}